test/JDBC/input/BABEL-3640.sql

```diff
@@ -0,0 +1,22 @@
+use master
```

Comment:
Could you create this as a form of MVU test? For example,

BABEL-3640-vu-prepare.sql
```
create database babel_3640_db;
use babel_3640_db;
create table ...
create proc babel_3640_p1 as
    ...
    EXEC sp_tables @P0, @P1, @P2, @P3;
```

BABEL-3640-vu-verify.sql
```
use babel_3640_db;
exec babel_3640_p1;
```

BABEL-3640-vu-cleanup.sql
```
drop ...
```

Add `BABEL-3640` in the latest schedule file.